app/helpers/container_summary_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ContainerSummaryHelper
   include TextualMixins::TextualName
-
+  include TextualSummaryHelper
```

Comment:
You can safely remove this (and the one in `vm_helper`), `TextualSummaryHelper` is included from `ApplicationHelper` and thus always available.